bundles/org.openhab.binding.kostalinverter/README.md

```diff
@@ -125,10 +125,9 @@ The following Channels are writeable
 |-------------------------------|---------------------------|-------------------------------------------------------|-----------------|    
 | chargeTimeEnd             | Number:Time     | Set charge time end, Value = 00:00  |    W    |
 | batteryType               | String          | Set battery type, Value = 1 = None, Value = 2 = PIKO Battery Li, Value = 3 = BYD B-Box HV  |   W    |
-| batteryUsageConsumption   | String          | Set battery usage consumption, Value = 100  |    W    |
+| batteryUsageConsumption   | String          | Set battery usage consumption level for power-consumption from battery, default value = 100 (W)  |    W    |
```

Comment:
If this is watts, `String` doesn't make sense here. You could make it `Number:Power`.